src/main/java/Task.java

```diff
@@ -0,0 +1,34 @@
+public class Task {
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public boolean isDone() {
+        return isDone;
+    }
+
+    public void setDone(boolean done) {
+        isDone = done;
+    }
+
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    public String toString(){
+        return "[" + getStatusIcon() + "] " + description;
+    }
```

Comment:
I like how you followed the naming conventions where Names representing methods must be verbs and are written in camelCase. Keep up the good work! :)